handler.py

```diff
@@ -3,7 +3,7 @@
 import requests
 #from SimpleHTTPServer import SimpleHTTPRequestHandler
 #import SocketHandler
-VOICEOSURL = "http://ec2-34-218-219-244.us-west-2.compute.amazonaws.com:5000/parse"
+RASASERVER_URL = "http://ec2-34-218-219-244.us-west-2.compute.amazonaws.com:5000/parse"
```

Comment:
Endpoint name was also changed - could have specified "Changed name of lambda function" instead of "Changed name of function".